### ISSUE TITLE
edit encrypted file in subfolder fails

### ISSUE THREAD

**Issue by melvyndekort** [twpayne/chezmoi#1094]
## Describe the bug

Editing an encrypted file that resides in a subdirectory fails.
Editing an encrypted file in the home directory works without issue.

## To reproduce

```
$ chezmoi add --encrypt .config/example.txt
$ chezmoi edit .config/example.txt
gpg: encrypted with 4096-bit RSA key, ID XXXXXXXXXXXXXXXX, created 2017-02-18
      "Example <user@example.com>"
chezmoi: stat /tmp/chezmoi-decrypted3070103435/dot_config: no such file or directory
```

## Expected behavior

I expect my editor to open with the unencrypted file.

## Output of command with `--verbose`

```
$ chezmoi edit --verbose .config/example.txt
gpg: encrypted with 4096-bit RSA key, ID XXXXXXXXXXXXXXXX, created 2017-02-18
      "Example <user@example.com>"
chezmoi: stat /tmp/chezmoi-decrypted578048408/dot_config: no such file or directory
```

## Output of `chezmoi doctor`

```
RESULT    CHECK                MESSAGE
warning   version              v2.0.2
ok        os-arch              linux/amd64
ok        config-file          /home/melvyn/.config/chezmoi/chezmoi.toml is a file
ok        source-dir           /home/melvyn/.local/share/chezmoi is a directory
ok        suspicious-entries   no suspicious entries found in /home/melvyn/.local/share/chezmoi
ok        dest-dir             /home/melvyn is a directory
ok        shell                found /bin/zsh
ok        editor               found /usr/bin/vim
ok        git-cli              found /usr/bin/git, version 2.31.0
ok        merge-cli            found /usr/bin/vimdiff
ok        gnupg-cli            found /usr/bin/gpg, version 2.2.27
warning   1password-cli        op not found in $PATH
ok        bitwarden-cli        found /home/melvyn/bin/bw, version 1.14.0
warning   gopass-cli           gopass not found in $PATH
ok        keepassxc-cli        found /usr/bin/keepassxc-cli, version 2.6.4
warning   keepassxc-db         not set
ok        lastpass-cli         found /usr/bin/lpass, version 1.3.3
ok        pass-cli             found /usr/bin/pass, version 1.7.3
warning   vault-cli            vault not found in $PATH
warning   secret-cli           not set
```

**Comment by twpayne** [comment]
Thanks for reporting this and for the clear and complete bug report :) This is the same issue as #1081 and was fixed in #1085.

I'll tag a new version including the fix later today.

**Comment by melvyndekort** [comment]
Sorry for creating a duplicate ticket!

But thnx for the amazing tool and quick fix!
